src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SkidSteerDriveSubsystem.java

```diff
@@ -6,30 +6,27 @@
 import com.systemmeltdown.robotlib.util.RobotMath;
 
 /**
- * Base class for any kind of "Skid Steer" drive base.
- * This makes assumptions that we will use encoders and velocity drive.
- * However, this makes zero assumptions about hardware or implementation of such.
+ * Base class for any kind of "Skid Steer" drive base. This makes assumptions
```

Comment:
Why are these formatting changes included in this PR? We should not be doing widespread formatting in the same PR as real code changes.